src/ui/components/Library/Team/View/NewTestRuns/TestRunSpecDetails.tsx

```diff
@@ -116,12 +117,26 @@ function ErrorGroup({
       className="flex w-full flex-col gap-2 overflow-x-auto rounded-md bg-[color:var(--testsuites-v2-error-bg)] px-3 py-4"
       data-test-id="TestRunSpecDetails-Error"
     >
-      <button className="flex flex-row gap-1" onClick={() => setExpanded(!expanded)}>
-        <div>({count})</div>
-        <div className="truncate">{summary}</div>
+      <button
+        className="flex flex-row items-start justify-between gap-1"
```

Comment:
Why did you go with `items-start` and an explicit `mt-0.5` on the items to align them vs using `items-center`?

Reply:
Good question, items-center wasn't getting the job done.

Reply:
Will address in a follow-on ticket where I clean up the CSS and move away from tailwind.